types/angular-animate/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: http://angularjs.org
 // Definitions by: Michel Salib <https://github.com/michelsalib>, Adi Dahiya <https://github.com/adidahiya>, Raphael Schweizer <https://github.com/rasch>, Cody Schaaf <https://github.com/codyschaaf>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 2.3
```

Comment:
`@types/angular` used to depend on `@types/jquery`. When I introduced `@types/jquery` v3, I added a TypeScript 2.3 requirement. That meant I had to add that requirement to all dependents which included `@types/angular` at the time. This was the breaking change that caused issues for people.

I later went and removed `@types/angular`'s dependency on `@types/jquery` and dropped `@types/angular`'s effective requirement to TypeScript 2.1.